one-ui/src/main/java/com/marklogic/hub/oneui/models/HubConfigSession.java

```diff
@@ -1002,4 +1008,19 @@ public void afterPropertiesSet() throws Exception {
         hubConfigImpl.hydrateConfigs();
         this.dataHub = new DataHubImpl(this);
     }
+
+    @Override
+    public void destroy() {
```

Comment:
There needs to be a test for this method. Otherwise, someone could add code that causes a null pointer exception or something similar, and we'd never know it. 

I think a suitable test would be to populate that map with two DatabaseClient objects, and then call destroy. Then verify that the map is empty (I can't think of a good way to verify that the clients were released). That's a decent smoke test.

Reply:
@rjrudin A new test class was created for checking the function.

Reply:
Just a note - this logic is the sort of thing that should eventually move into a HubClient class, where it'd be a lot easier to test - i.e. create a HubClient with 3 DatabaseClients, then call destroy, then verify that each of the 3 DatabaseClients throws an error if you try to do something with it.